partner/app/main.py

```diff
@@ -81,7 +81,9 @@ async def read_tilesp(
     country_code: str = Query(
         ..., alias="country-code", example="US", length=2, regex="^[A-Z]{2}$"
     ),
-    region_code: str = Query(..., alias="region-code", example="NY"),
+    region_code: str = Query(
```

Comment:
Nit: let's add a comment to note that `country_code` and `region_code` follow [ISO 3166-1 Alpha 2](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2) and [ISO 3166-2](https://en.wikipedia.org/wiki/ISO_3166-2), respectively.

Reply:
yes, that's a good idea. I made the changes. Thanks for the review @ncloudioj ! :)